sound/soc/codecs/rt1308-sdw.c

```diff
@@ -123,6 +123,10 @@ static int rt1308_read_prop(struct sdw_slave *slave)
 	unsigned long addr;
 	struct sdw_dpn_prop *dpn;
 
+	prop->scp_int1_mask = SDW_SCP_INT1_IMPL_DEF | SDW_SCP_INT1_BUS_CLASH |
+		SDW_SCP_INT1_PARITY;
```

Comment:
I don't think rt1308 needs SDW_SCP_INT1_IMPL_DEF, but needs @shumingfan's confirm and can remove it along with rt1308_interrupt_callback() in another commit.

Reply:
Good catch @bardliao, it's the same with most amplifiers indeed.

Reply:
done.